pytorch_lightning/metrics/classification/accuracy.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 from typing import Any, Callable, Optional
 
+from deprecate import void
```

Comment:
Why does this come from your deprecate package @Borda ? This doesn't look like a good fit for me. Can we move it to our trips here since it's not about depreciation?

Reply:
what do you mean by trips? moved to our utils...

Reply:
That's what I meant. Was autocorrect, sorry :D I really need to turn off the automatic changes